### ISSUE TITLE
Motion to drop gist:Artifact

### ISSUE THREAD

**Issue by marksem** [semanticarts/gist#385]
I am finding Artifact not useful, and pretty redundant.  I.e., most (if not all?) domain concepts I need to "ground" in an upper class seem to find a home under other concepts. Feels like Artifact is an aspect-oriented class that we don't need.

**Comment by ungricht** [comment]
I object to the motion on the grounds that I'm using it for my project metadata ontology.  Here's a snippet of my work.  Happy to discuss in an Issue Review Meeting. 
![image](https://user-images.githubusercontent.com/7063526/95110148-99205600-070b-11eb-8d1d-7b4631e9051e.png)

**Comment by uscholdm** [comment]
Its a good candidate to get the boot, since its not about what the thing is, but how it came about.  That said, what will you do with the newly-to-be orphaned classes such as `Component`, `Network` and `System`? I am also using it for other things for Platts:

![image](https://user-images.githubusercontent.com/10777545/95145102-29bc5d80-072f-11eb-9353-340122bc2304.png)

QUESTION: can this image be made smaller?

**Comment by marksem** [comment]
Component feels like a role things, i.e. how something is used.  I.e., any hardware component would seem to be Equipment.  Any software component would seem to be Intellectual Property.

A data set sounds like a type of collection to me.
So does a network... any collection of related things. 

Software Application is Intellectual Property (a work, an invention).

System is somewhat ambiguous, as I've discovered at MS.  Which do we mean...
> 1. a set of things working together as parts of a mechanism or an interconnecting network.
> "the state railroad system"
> 2. a set of principles or procedures according to which something is done; an organized scheme or method. "a multiparty system of government"
1 sounds like a collection (a set of things working together for some purpose)
2 sounds also like a collection, but it is a set of principles/procedures.

VersionedItem sounds like a role as well.... if X has version, it is a Versioned Item.  That's like a class called NamedItem. where anything with a name goes into it.  We can define such classes (indeed OWL is great for this), but in gist itself we usually don't.

**Comment by rjyounes** [comment]
I agree that component and versioned item are relationships rather than things. 

Perhaps this is tangential, but I would say versioned item is a relationship rather than a role - e.g., the versioned item is a versionOf the unversioned one. A role, unlike a relationship, is generally _in_ something larger, whereas a relationship is not, or not necessarily. E.g., the _role_ of patriarch in a family vs the father/child _relationship_. But I agree it's not a type of thing.
In that sense I think component is also a relationship. If you describe what the component _does_ within a larger activity, that would be  a role. E.g., a keyboard is a component of a laptop (relationship), but its role or function is to allow data entry.  

A network is more than just a collection of related things. My cutlery set is a collection of eating utensils, but it does not comprise a network. Merriam-Webster says "an interconnected or interrelated chain or system..." which I think is getting close. Network seems like a subclass of System.

DataSet as a type of Collection sounds good to me. It is also a type of IntellectualProperty.

Having said that, I'm not completely convinced by the premise that Artifact shouldn't be a class because it's defined by how things came about rather than what they are. The man-made vs naturally-occurring distinction seems rationale and worthy of capturing. Further, in many of our client ontologies we define classes of things based on their relationships to other things. E.g., Patient or Employee are not really types of things in the same way that Person or Tree is - they are Persons in certain relationships to other objects. Yet in some cases we find it convenient to define such classes with restrictions to provide the specific meaning. I don't see a fundamental difference between that and a class Artifact which has a madeBy relationship to some human or collection of humans. (Whether it is good modeling to define such classes at all is still an open question in my mind, but it is accepted practice at SA.)

**Comment by uscholdm** [comment]
@rjyounes @marksem 
What is the status of this?

**Comment by jtdelany58** [comment]
See also #110 #111 #548

**Comment by rjyounes** [comment]
Looks to be still unresolved. I'll add it to the current project so we can discuss it. 

The other issues @jtdelany58 referenced are already closed, so not relevant.

**Comment by johnwcowan** [comment]
Public-domain (i.e. government-made, explicitly abandoned, etc.) software is not any sort of property.

**Comment by johnwcowan** [comment]
Public domain intellectual property is definitely property.  In the U.S., the work product of government employees is in the public domain in the U.S., but not in other countries, where it belongs to the U.S. government.

Public domain real estate is U.S. government property managed by the Bureau of Land Management.  It includes grazing rights (mostly in the West) and mineral rights that have been severed from surface rights.  It too is property, as it can be privatized at any time.

**Comment by rjyounes** [comment]
`Building` and `Equipment` have other superclasses.

These would end up at the top-level:
* `System`
* `Network`
* `Content`
* `IntellectualProperty`
* `Component`

**Comment by uscholdm** [comment]
> I would argue that Network should have never been an Artifact. A network of rivers or blood vessels or nerves are not "person-made". I would argue same for Components and Systems. So even if we didn't remove Artifact, those 3 would probably need to go to root level anyway.

Is the concrete proposal to drop Artifact and move its top level subclases to be orphan classes?

Isn't Network being removed anyway?
I'm not sure we want the class, Component in gist proper. Its a role of a thing.

**Comment by marksem** [comment]
I am working on a detailed response on what might move up. But it is a WIP. :'-(
________________________________

**Comment by marksem** [comment]
Thoughts so far:

Component is a role, drop it
 used in: 
  - Definition of System
 how to remove:
  - Delete Component
  - Define system in terms of having parts and a Behavior or function

System seems top level

Network: is it necessarily a System by our definition? It must have parts.  Does it always have some goal or behavior?  

Function is an intention by our definition.  But dictionary says: 
 "an activity or purpose natural to or intended for a person or thing"
Updating the definition of function to be the above would allow things like Systems and Networks to have Functions, without them being man-made.
E.g.:, does the atmosphere have functions?  Are those functions intentions?

But we could not always say hasGoal Function, as goal implies Intention. 
(Perhaps hasGoal should have range Intention? :O )

Do we need both Content and IntellectualProperty?  
I don't think IntellectualProperty need be (nor is intended to be) the legal definition in say the US.  It just means "A work, invention or concept, independent of its being expressed in text, audio, video, image, or live performance.  IP can also be tacit knowledge, know-how, or skill. Also includes Brands."

**Comment by uscholdm** [comment]
> Do we need both Content and IntellectualProperty?

I think so, given not all content is intellectual property and the the importance of the latter in any enterprise, especially form a legal perspective. A more interesting question, perhaps, it whether IntellectualProperty is a subclass of Content. The definition of ContentExpression hints that this is so:
- `Intellectual Property reduced to text, audio etc.  If it contains text (written or spoken), it may be in a language`

**Comment by uscholdm** [comment]
> Component is a role, drop it

Agree

**Comment by rjyounes** [comment]
IntellectualProperty is not a subclass of Content. Our definition reads:

A work, invention or concept, independent of its being expressed in text, audio, video, image, or live performance.  IP can also be tacit knowledge, know-how, or skill. Also includes Brands.

I think the definition of ContentExpression should be modified to read "Some types of intellectual property..."

Common definitions of intellectual property rights include patents, trademarks, copyrights, trade secrets, industrial designs, geographical indications, and plant breeds.

**Comment by rjyounes** [comment]
> Network: is it necessarily a System by our definition? It must have parts. Does it always have some goal or behavior?

A network of, say, rivers has no goal or behavior.

**Comment by uscholdm** [comment]
> I think the definition of ContentExpression should be modified to read "Some types of intellectual property..."

Most of the examples of intellectual property are ultimately rendered as content, e.g. patents, books etc.  Are there examples of intellectual property that are not?  Brand is an example that comes to mind.

**Comment by uscholdm** [comment]
> A network of, say, rivers has no goal or behavior.

A related example is our network of veins and arteries, which one may argue there is no overt goal or intention but there is definitely a biological function.  We impute goals and intentions and functions as if they existed from the beginning.  We say water 'wants to run downhill', for example. A scientist can no doubt rattle off one or more 'functions' of the water cycle which includes networks of rivers emptying into the sea.

In summary, there is a difference between 
- a goal that is explicitly had by a human
- an imputed goal that appears to be a goal or function of something but involves no human intention

**Comment by MichaelSullivanArchitect** [comment]
regarding Brand. Yes it is an abstraction, yet the majority of formal brands (as opposed to those social media types who claim their own "brnad") are in fact well-documented, either with official list of colors, fonts, layouts, logos, etc.

**Comment by rjyounes** [comment]
> In summary, there is a difference between a goal that is explicitly had by a human and an imputed goal that appears to be a goal or function of something but involves no human intention

Excellent point. Is `gist:hasGoal` meant to apply to either, or only to human intention?

**Comment by rjyounes** [comment]
> Most of the examples of intellectual property are ultimately rendered as content, e.g. patents, books etc.  Are there examples of intellectual property that are not?  Brand is an example that comes to mind.

Performance art, plant breeds. I'm not sure how plant breeds are protected but a written description is not the breed itself.

**Comment by rjyounes** [comment]
Seems like `Network` could be a subclass of `Collection`.

**Comment by uscholdm** [comment]
> Seems like `Network` could be a subclass of `Collection`.
I think that the network is much more than just a collection. What exactly is it a collection of? Fundamentally there are a bunch of nodes and there are a bunch of links between them. I don't see how calling it a collection is helpful.

**Comment by rjyounes** [comment]
> I  think that the network is much more than just a collection. What exactly is it a collection of? Fundamentally there are a bunch of nodes and there are a bunch of links between them. I don't see how calling it a collection is helpful.

Clearly it is much more - that's why it would be a specialization. The advantage of subclassing is to reduce the number of classes at the top level. A bunch of nodes plus a bunch of links sounds like a collection to me.

**Comment by uscholdm** [comment]
If its a collection, then it should be natural to think of a network as having members and using gist:isMemberOf to make the link.  What do you see as the members of a network?

To me, it is natural to think of the network is having components (rather than members) which would include nodes and links.

**Comment by rjyounes** [comment]
* Delete `Artifact` 
* Delete `Component`
* Move to top-level:
  *  `Content`
  * `IntellectualProperty`
* Move `System` and `Network` under `Collection`
* `Building` and `Equipment` remain subclasses of `PhysicalIdentifiableItem`
